kernel/kernel-impl/src/main/java/org/sakaiproject/memory/impl/BaseMemoryService.java

```diff
@@ -56,8 +58,8 @@
     public void init() {
         log.info("INIT");
         if (memoryService == null) {
-            // defaults - ehcache (new)
-            String cacheManagerType = TYPE_EHCACHE;
+            // defaults - hazelcast for limited use and ehcache for the rest
```

Comment:
By default, it will attempt to fire up Hazelcast for caching BaseUserEdits. Everything else still uses Ehcache. Even if specified as TYPE_HAZELCAST, Hazelcast is only used for the one type.